gamefixes-steam/1277930.py

```diff
@@ -0,0 +1,11 @@
+""" Game fix for Riddle Joker
+"""
+#pylint: disable=C0103
+from protonfixes import util
+def main():
+    """
+    """
+    util.protontricks('quartz')
+    util.protontricks('wmp9')
+    util.protontricks('directshow')
+    util.disable_protonaudioconverter()
```

Comment:
In both the latest GE-Proton and UMU-Proton, for Riddle Joker, the intros play just fine after applying `quartz`, `wmp11` and `qasf`. This suffices and will likely be successful for the other games and their ending videos as well, given that they use the same engine -- KiriKiri.

In a bit, I will merge these fixes and be sure to credit your contribution. No need to reupload.

Reply:
Alright, finally its done.